plugin-oar2/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "plugin-oar2"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+alumet = { version = "0.4.1", path = "../alumet" }
+anyhow = "1.0.86"
+notify = { version = "6.1.1", default-features = false, features = ["macos_fsevent"] }
```

Comment:
It does nothing on non-macOS targets, and selects a particular "backend" on macOS. When performing local tests, Sofia found that this one was better than the regular backend, which didn't report folder events properly on the macOS version she was using.

Reply:
And the reason we need to specify a backend manually is because we use `default-features = false` in order to disable the `crossbeam-channel` feature, which causes interferences with tokio. See https://docs.rs/notify/latest/notify/#crossbeam-channel--tokio

Reply:
thank you for your explanations, it’s clear and ok for me